### ISSUE TITLE
Improve accessibility (e.g. alt text) for images

### ISSUE THREAD

**Issue by brainwane** [SolutionGuidance/psm#509]
To better serve people who use screenreaders, we should ensure that the `img` element's `alt` text serves the same purpose and presents the same information as the image. In `pa11y` this is `Section508.A.Img.GeneralAlt`. Examples where we currently do not comply with this (mostly with modal icons) are in comments below.

[Here is a guide to writing good alt text](https://denise.dreamwidth.org/2013/01/31/writing-alt-text.html).

**Comment by jasonaowen** [comment]
I am surprised, as those `img` elements *do* have `alt` text, and the name of the module seems to me like a reasonable substitute for the logo of the module. @brainwane, can you say more about why these are examples of bad behavior? What is the right thing to do here?

**Comment by brainwane** [comment]
@jasonaowen I'm wondering about the possible over-escaping (some of these slashes seem unnecessary) causing difficulties for screenreaders and thus causing pa11y to flag problems.

**Comment by brainwane** [comment]
Also we need to fix places where the `img` element is marked so that it is ignored by assistive technology (`Section508.A.Img.Ignored`) or where it's missing an `alt` attribute altogether (`Section508.A.Img.MissingAlt`). Stuff like:
~~~
<img src=\"Provider-Statement_files/icon-x.png\" alt=\"\">
<img src=\"License-Information_files/icon-print2.png\" alt=\"\">
<img src=\"Summary-Information_files/icon-print.png\" alt=\"\">
<img id=\"__lpform_firstName_icon\" src=\"data:image/png;base64,iV[...]
~~~

**Comment by jasonaowen** [comment]
> I'm wondering about the possible over-escaping (some of these slashes seem unnecessary) causing difficulties for screenreaders and thus causing pa11y to flag problems.

I'm not seeing those escape characters in the rendered HTML on my local application, and I think if they were present they'd be syntax errors. Do you see them on the live application? Are they maybe an artifact of saving a copy locally? Or with how pally reports?

**Comment by brainwane** [comment]
I think there's something about `pa11y`'s CSV export that caused those slashes. The actual local saved pages have code like `<img src="Enrollment-review_files/psm-logo.png" alt="Provider Screening Module">` which is much more sensible.

**Comment by cecilia-donnelly** [comment]
@PaulMorris has found that these "no alt text" errors refer to icons (as @brainwane mentioned earlier).  Currently, we plan to use CSS to show these icons, instead of inserting them as images.
